content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/accordion.html

```diff
@@ -13,32 +13,30 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
-
 <div data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
-     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
      class="cmp-accordion"
-     data-cmp-is="accordion">
-    <sly data-sly-test="${accordion.items && accordion.items.size > 0}"
-         data-sly-repeat.item="${accordion.items}">
-        <div class="cmp-accordion__item${item.name == accordion.expandedItem ? ' initially-expanded' : ''}"
-             data-cmp-hook-accordion="item"
-             aria-expanded="false"
-             role="button"
-             tabindex="0">
-            <h3 data-sly-element="${accordion.headingType @ context='elementName'}"
-                class="cmp-accordion__item__title"
-                role="heading">
-                ${item.title}
-                <span class="cmp-accordion__item__title__icon"></span>
-            </h3>
-        </div>
-
+     data-cmp-is="accordion"
+     data-cmp-single-expansion="${accordion.singleExpansion}">
+    <div data-sly-test="${accordion.items && accordion.items.size > 0}"
```

Comment:
That's cool that we have a wrapper for each item now :) Just wondering, maybe these should be `<li>` tags inside and unordered list now?

Reply:
Does it add value? If not, I think `<div>` are cleaner - they don't require any CSS reset.

Reply:
Well true, on a second thought I will agree.